src/db_migrations.rs

```diff
@@ -80,3 +82,37 @@ pub fn migrate(conn: &mut Connection) -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate tempdir;
+
+    use self::tempdir::TempDir;
+    use super::*;
+
+    #[test]
```

Comment:
Note: these tests reproduced the problem I was seeing when I tried to deploy the latest octobot. :)